themes/adminimal_medela/build/js/publication-rules.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file
+ * Custom functionality for publication rules.
+ */
+(function ($, Drupal, window) {
+
+  'use strict';
+
+  Drupal.behaviors.adminimalMedelaPublicationRules = {
+    attach: function (context, settings) {
+
+      $(".listing-embedded-filters--content-types").each(function (i) {
+
+        var showPub = true;
+        $(this).find("input:checkbox").each(function (i) {
+          let label = $(this).val();
```

Comment:
Best practise is to not use duplicate instances of `$(this)` in the same code block. Each time one does this jQuery initialises, and has to manage, a separate instance of the same element . Also better to specifically refer to elements rather than use `this`. Makes it easier when converting code to ES6.

Instead consider initialising the object once and then reusing it:

```
const $element = $(this);

let label = $element.val();
let status = $element.is(":checked");
```

Or even better:

```
$(".my-element").find("input:checkbox").each(function (i, element) {
  const $element = $(element);
  ...
});
```